### DISCUSSION TITLE
Score for DMARC soft fail

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#4297] [General]
Friends,

Just wondering if DMARC soft fail isn't too high of a score. 

My point if, many don't know about DMARC from header x sender misalignment and this score is quite high, 7 points, our threshold is 11.

DMARC_POLICY_SOFTFAIL(7)

So what happens is, I see many emails where people send newsletters via Amazon, etc. 

Example, FROM header

From: Entrega The Body Shop <pesquisaentrega@thebodyshop.com.br>

SENDER/MAIL FROM: bounces+2445086-0edc-flavio=lucanet.com.br@sendgrid.net

As you can see in the attached image, they have P = NONE, policy none. Many, surely they, probably imagine nothing is wrong.

But there's a misalignment. It's not crystal clear the error from their record. See what I mean? Its not like SPF where you clearly see a non-declared IP.

Besides, DMARC isnt a technology that is widely used, there are still a few shortcomings.

I would lower the score a bit, maybe 3, 4 points.

My 2 cents! :)

Thanks!

![image](https://user-images.githubusercontent.com/44004645/194396795-9f9a96f9-df24-4e14-8a18-1f8b88024b7d.png)

**Comment by vstakhov** [comment]
The default score of this symbol is `0.1`...

**Comment by flaviozluca** [comment]
Sorry for that, friend. My sysadmin changed the score, I made confusion.

Thank you.
